pkg/action/dependency.go

```diff
@@ -45,7 +45,7 @@ func NewDependency() *Dependency {
 }
 
 // List executes 'helm dependency list'.
-func (d *Dependency) List(chartpath string, out io.Writer) error {
+func (d *Dependency) List(chartpath string, out io.Writer, columnWidth uint) error {
```

Comment:
changing the function signature would be construed as a backwards-incompatible change, so this will have to be changed. For context: https://github.com/helm/helm/blob/main/CONTRIBUTING.md#semantic-versioning

In other actions, we add new fields to the action object. Those are then read during `Run`. That way we can provide these features without breaking compatibility.

For example, take a look at install.go: https://github.com/helm/helm/blob/43853ea772e779b617b011a8d4e69205636fa4f9/pkg/action/install.go#L68-L104